rmgpy/molecule/adjlist.py

```diff
@@ -524,11 +524,17 @@ def from_adjacency_list(adjlist, group=False, saturate_h=False, check_consistenc
             multiplicity = int(line.split()[1])
         if len(lines) == 0:
             raise InvalidAdjacencyListError('No atoms specified in adjacency list: \n{0}'.format(adjlist))
-
+        
```

Comment:
Can you run an auto-formatter to remove all these unnecessary blanks?

Reply:
I think it's inappropriate to run the auto-formatter on RMG files that are commonly modified, because I think that will create very large nasty merge conflicts if anyone modified the file and tries to rebase?

Reply:
There are tools like "[darker](https://pypi.org/project/darker/)" (that runs "[black](https://github.com/psf/black)" only on lines of code that you modified). Maybe there are others. Anyway, I agree please don't run something wholesale across the whole file, especially during a topic pull request.    But we can try to avoid making it worse :)

Reply:
Besides what @rwest mentioned, if you use VSCode, there are options where you can choose only run auto-formatter on the lines you modified as well!